egs2/fsc_challenge/asr1/README.md

```diff
@@ -0,0 +1,35 @@
+<!-- Generated by ./scripts/utils/show_asr_result.sh -->
+# RESULTS
+
+## Environments
+- date: `Sun Oct  3 22:25:25 EDT 2021`
+- python version: `3.8.11 (default, Aug  3 2021, 15:09:35)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.3a3`
+- pytorch version: `pytorch 1.9.0+cu102`
+- Git hash: `a1a55e1eef2a74d2b8580d8071ce5229e7fa654c`
+  - Commit date: `Mon Nov 8 23:56:06 2021 -0500`
+- Pretrained Model
+  - Zenodo: https://zenodo.org/deposit/5656007
```

Comment:
How about adding a model to HF hub as well and add an HF hub link?

Reply:
Will do!